### DISCUSSION TITLE
Feature:  check for duplicates

### DISCUSSION THREAD

**Discussion by hollisticated-horse** [donuts-are-good/bearclaw#22] [Bearclaw Discussions]
**describe the feature**
Currently, when generating a new blog post, then updating the file name:

`mypost.md` -> `myPost.md`

will generate a new .html file, but the old one stays. 

Add checking for post duplicates

- for example fuzzy search for similar.html files, maybe just move them to a triage folder so that the user can review and remove if need be ? 
- Or just check if a file is referenced/linked ?

**Comment by donuts-are-good** [comment]
I want to acknowledge that this has been seen, however I'm not fully sold on it yet, though I'd like to see how this idea takes shape. The first case of fuzzy searching might be too complex to be a good fit, but I'd like to ask a question about the case where upper/lower is altered:


In theory, how would you handle the remainder file with the old spelling both for the file in the filesystem and how should that affect the listing on Posts? (for example, titles appended with UPDATE!, or similar)

**Comment by donuts-are-good** [comment]
Moving to discussions to give it some room to develop
